src/components/CreateProject/ProjectForm/CreateProjectForm.tsx

```diff
@@ -22,10 +24,42 @@ const FormSchema = yup
   })
   .required();
 
+// NOTE: the spread is to avoid the type error with notification props not accepting data attribute
+const notifySuccessCreation = (name: string) =>
+  showNotification({
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    ...{ 'data-test': 'notification-success' },
+    color: 'successDefault',
+    title: messages.projects.create,
+    message: `پروژه با نام ${name} با موفقیت ساخته شد`,
+  });
+
+const notifyFailedCreation = (name: string) =>
+  showNotification({
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    ...{ 'data-test': 'notification-fail' },
+    color: 'errorDefault',
+    title: messages.projects.create,
+    message: `مشکلی در مرحله ایجاد پروژه ای بانام ${name} به وجود آمده است. لطفا دوباره تلاش کنید`,
+  });
+
 export const CreateProjectForm = ({ dismiss }: Props) => {
-  const onSubmit = React.useCallback(({ name, description }: FormSchema) => {
-    console.log('name:', name, 'description:', description);
-  }, []);
+  const [createProject] = useCreateProjectMutation();
+
+  const onSubmit = React.useCallback(
+    ({ name, description }: FormSchema) => {
+      createProject({ variables: { input: { name, description } } })
+        .then(({ data }) => {
+          console.log(data);
+          notifySuccessCreation(name);
```

Comment:
I suggest to show the responded name `data. ... .name`

Reply:
> `Sees the successful notification when Submits the form correctly` why this test didn't pass? because of backend problems?

about this
yes it is because of that but IMO there is still something missing maybe
like a timeout or something (of course the current timeout is not bad though it's 4s I think) 
but yeah the reason that it's not passing is that

Reply:
> I suggest to show the responded name `data. ... .name`

we can do that but cause we're doing this in the console, having the whole object there for debugging purposes is good, don't u think?